modules/metrics/statistic.go

```diff
@@ -0,0 +1,294 @@
+package metrics
+
+import (
+	"code.gitea.io/gitea/models"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const namespace = "gitea_"
+
+// Collector implements the prometheus.Collector interface and
+// exposes gitea metrics for prometheus
+type Collector struct {
```

Comment:
Maybe this file should be named "collector.go" instead to reflect the content better? :)